src/commands/branch.rs

```diff
@@ -211,7 +219,10 @@ fn find_globs(view: &View, globs: &[String]) -> Result<IndexSet<String>, Command
             .branches()
             .iter()
             .map(|(branch_name, _branch_target)| branch_name)
-            .filter(|branch_name| glob.matches(branch_name))
+            .filter(|branch_name| {
+                glob.matches(branch_name)
+                    && (allow_deleted || view.get_local_branch(branch_name).is_some())
+            })
```

Comment:
No need to look up branch targets again.
Reorder `.filter()` and `.map()` (or use `.filter_map()`?)

Reply:
Thanks!